mne/decoding/transformer.py

```diff
@@ -715,3 +717,164 @@ def transform(self, X):
         X = self.estimator.transform(X)
         X = np.reshape(X.T, [-1, n_epochs, n_times]).transpose([1, 0, 2])
         return X
+
+
+class Filterer(TransformerMixin):
```

Comment:
'Filterer'? what an awkward name and not a proper English word. The verb is 'to filter'. How about just calling it 'Filter', or better yet, 'BandpassFilter'?